apps/docs/scripts/update-algolia-index.ts

```diff
@@ -273,8 +287,9 @@ async function updateAlgoliaIndex() {
 					'asc(articleIndex)',
 					'asc(headingIndex)',
 				],
-				distinct: 4,
+				distinct: 3,
 				attributeForDistinct: 'section',
```

Comment:
idk if you've played with removing these two entirely but it could be worth a look (although a bit weird having results from different sections interleaved)

Reply:
Maybe! Will probably revisit, but this feels like a small drive-by improvement for now